<launch>
  <arg name="robot_ip" default="192.168.10.14" />
  <arg name="sensor_port" default="63351" />

  <node name="robotiq_ft_streamer" pkg="robotiq_ft_sensor" type="robotiq_ft_streamer.py" output="screen">
    <param name="robot_ip" value="$(arg robot_ip)" />
    <param name="sensor_port" value="$(arg sensor_port)" />
  </node>
</launch>
